Description: Fixes spell
Author: Herbert Parentes Fortes Neto <hpfn@ig.com.br>
Last-Update: 2015-10-08
Index: openresolv-3.8.0/resolvconf.8.in
===================================================================
--- openresolv-3.8.0.orig/resolvconf.8.in
+++ openresolv-3.8.0/resolvconf.8.in
@@ -128,7 +128,7 @@ on the
 Here are some options for the above commands:-
 .Bl -tag -width indent
 .It Fl f
-Ignore non existant interfaces.
+Ignore non existent interfaces.
 Only really useful for deleting interfaces.
 .It Fl m Ar metric
 Set the metric of the interface when adding it, default of 0.
